parser/src/parser/grammar.lalrpop

```diff
@@ -118,6 +120,40 @@ PeriodicColumn: PeriodicColumn = {
         PeriodicColumn::new(name, values),
 }
 
+// RANDOM VALUES
+// ================================================================================================
+
+RandomValues: RandomValues = {
+    "random_values" ":" <rand_values: ListElems<RandValuesDecl>> =>? match rand_values.len() {
+        1 => Ok(rand_values[0].clone()),
+        _ => Err(ParseError::User{
+            error: Error::ParseError(
+                InvalidRandomValues("No more than one set of random values can be declared".to_string())
+            )
+        })
+    }
```

Comment:
Can this not be replaced with just
`"random_values" ":" <rand_values: RandValuesDecl> => rand_values`?
Or was your idea to do better error reporting here? If you change it, then `ListElems` definition below can also be removed.

Reply:
Yes, I was trying to implement check of the case when more than one set of random values has been declared. If I leave it as it was (and this is exactly what you suggested) when it will panic with `UnrecognizedToken` error, and I wanted to handle this error with specific message. I don't really like the implementation, but I haven't been able to come up with anything better.

Reply:
I think we should be able to do something like:
```
"random_values" ":" <rand_values: RandValuesDecl+>
```
and then return an error if length is not one.
Alternatively, we could do:
```
"random_values" ":" <rand_values: RandValuesDecl*>
```
so that we can report better error for empty random values section as well.

Reply:
Although we should do better error reporting for other things in the grammar as well.

Reply:
Indeed, that could be done much easier, by bad. I think that `+` macro should be enough, because `rand: *nothing*` is just an incorrect declaration in general and will cause parsing error, and `rand: []` is handling in `RandValuesDecl` parsing

Reply:
Yeah, makes sense.